### ISSUE TITLE
Tekken 7 Controller doesn't work.

### ISSUE THREAD

**Issue by MeGA-ct** [ValveSoftware/Proton#1119]
Hi, I have an issue with Tekken 7.

The announcement says that it's supported fully, but for me the controller doesn't work at all.

The controller it's recognized by the system but in the game it just don't respond at all.

https://gist.github.com/MeGA-ct/8d1413f34790f24649461aaf84fcf02b

If you need any more info let me know.

**Comment by kadogo** [comment]
@MeGA-ct 
I'm not sure it will help you but in my case it work for my Xbox controller.
I need to launch the game in big picture but without Xbox support.

![screenshot at 11-42-20](https://user-images.githubusercontent.com/5759396/44944711-229e2500-addc-11e8-8c60-d4e370efb045.png)

I hope it will help you.

**Comment by MeGA-ct** [comment]
@kadogo Thx for the reply, but i don't think this will work for **Steam Controller**, maybe for Generic controller that i have.

**Comment by kadogo** [comment]
Yes with steam controller it's already unticked I think so it will not change I think

**Comment by MeGA-ct** [comment]
@kadogo Well at least the workaround for other controllers that are not Steam controller works.

The issue still stands for Steam Controller.

**Comment by ZereoX** [comment]
Hey @MeGA-ct, did you setup the custom UDEV rules for the Steam Controller? I found that Gamepad emulation won't work otherwise.

See this link: https://steamcommunity.com/app/353370/discussions/0/490123197956024380/

PS: Make sure you change the "GROUP=" on Line 4 to something you are part of.

**Comment by MeGA-ct** [comment]
Update:
No **udev** rules for steam controller, I have the **steam-devices** package installed.
The workaround seems to work, with Steam controllers now too.

If I deactivate all the configuration support the Steam and Generic controller works.
![imagen](https://user-images.githubusercontent.com/3133372/44958540-ee149100-aee1-11e8-8b0f-a9ad1a263d4e.png)

**Comment by syedMujeebUlHasan** [comment]
For me it fixed when I mapped my xbox360 button to same as xbox one controller button, tekken7 is registering my xbox one controller as xbox one and xbox360.
